.github/workflows/performance-test.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Gatling Performance Tests
+
+on:
+  push:
+    branches:
+      - dev
+      - iv-perf-devbase
+    tags:
+
+jobs:
+  performance-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      # See master_push.yml
+      - name: Get Vault token
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Grant execute permission for render_config
+        run: chmod +x render_config.sh
+      - name: Render configuration for tests
+        run: ./render_config.sh ${{ steps.vault-token-step.outputs.vault-token }} dev
+      - name: Ensure read permissions for rendered files
+        run: sudo chmod +r src/test/resources/rendered/wsm-firecloud-account.json
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Run Gatling performance tests
+        env:
+          WSM_BASE_URL: https://workspace.ichang.integ.envs.broadinstitute.org/
+          GATLING_CONCURRENCY: 10
+          CICD: GHA
+          FIRECLOUD_SERVICE_ACCOUNT_PATH: ../src/test/resources/rendered/wsm-firecloud-account.json
+        run: |
+          cd terra-workspace-manager-performance
```

Comment:
This is to make sure that the `build.gradle` of module / subproject `terra-workspace-manager-performance` is loaded rather than the root project's `build.gradle`. We structured the configuration so that the load test framework is defined in the subproject's `build.gradle` (in this case Gatling framework). In the future, when we develop perf tests using say Jmeter framework, we can update the subproject's `build.gradle` to Jmeter framework and configured the source sets to load tests from say src/test/jmeter within the subproject.